packages/wp-comments/src/__tests__/mocks/helpers.ts

```diff
@@ -0,0 +1,30 @@
+import { ResponseInit, Response as NodeResponse } from "node-fetch";
+
+/**
+ * Utility to mock responses from WP REST API.
+ *
+ * It instantiates a `node-fetch` response object using the passed information
+ * with the standard `Response` type.
+ *
+ * @param body - Body of the `Response` object.
+ * @param init - Other `Response` properties.
+ *
+ * @returns `Response` object.
+ */
+export const mockResponse = (body, init?: ResponseInit): Response =>
```

Comment:
This is just a comment for a future improvement: We could have a sort of test-utils package because there already exists the same function in the wp-source (twice in fact, once in helpers, other in populate.tests.ts 😅 )